packages/example/bin/build.js

```diff
@@ -1,28 +1,29 @@
 #!/usr/bin/env node
 
+/* eslint-disable no-console */
```

Comment:
Did we need this? I don't see any `console.log` added here.

Reply:
see lines 31 & 34